upup/models/cloudup/resources/addons/aws-ebs-csi-driver.addons.k8s.io/k8s-1.17.yaml.template

```diff
@@ -352,7 +352,7 @@ spec:
             - --endpoint=$(CSI_ENDPOINT)
             - --logtostderr
             - --k8s-tag-cluster-id={{ ClusterName }}
-            - --extra-tags=KubernetesCluster={{ ClusterName }}
+            - --extra-tags=KubernetesCluster={{ ClusterName }},{{ CloudLabels }}
```

Comment:
If `cloudLabels` is empty, will the trailing comma be a problem?

Reply:
That's what I'v verifying in exactly this moment as I realised this as well after re-reviewing the PR.

Reply:
Maybe you can add `ClusterName` to `CloudLabels` and will always be ok.

Reply:
Verified: It works just fine with the trailing comma.

If still required, I can try to avoid that trailing comma. I checked if any ifs are used inside the templates and it looks like you've successfully avoided having any ifs/range, ... in the templates and I did not want to be the first one introducing it 🙈

Reply:
@hakman That sounds good, but then I'd like to rename the `CloudLabels` template var as it does not resemble the `cloudLabels` from the cluster spec anymore. I'll call it "EbsCsiExtraTags" then.

Reply:
@olemarkus Any thoughts?

Reply:
@hakman Done. I called it `CsiExtraTags` just in case other clouds/addons might also want to use it in the future. I've also squashed everything into the initial commit.

Reply:
Maybe rename it to `CsiExtraTags`, to match the flag name.
This should be bumped to `0.8.0-kops.2` also:
https://github.com/kubernetes/kops/blob/cb1edd047eaa683f1411a6dd600ee2c4cd7bf6a8/upup/pkg/fi/cloudup/bootstrapchannelbuilder/bootstrapchannelbuilder.go#L1024

Reply:
I am good. We can always make the template functions more reusable later when there is need.

Reply:
@hakman Fixed the name and bumped the version.

Reply:
Thanks. Please also squash the commits and this should be it.

Reply:
Done